### ISSUE TITLE
Doesn't work on my computer

### ISSUE THREAD

**Issue by mll0** [EthanShoeDev/XPSThermalTray#3]
Thanks for this software. It's the workaround I need because in "Optimized" mode the fans are very noisy even with low stress...

Anyway, I'm on a Dell Latitude 5480 (_not_ an XPS I'm aware), and the tray app loads, but if I select "Quiet" nothing happens, end eventually, the tray icon disappears and the application silently quit.

I come from https://www.reddit.com/r/DellXPS/comments/132gzs6/seeking_help_with_programmatic_control_of_dell/, and when I test access to ThermalManagement using the path you provided, I get:

```
PS C:\Windows\system32> cd dellsmbios:
PS DellSmbios:\> Get-Item .\PreEnabled\ThermalManagement
Get-Item : Attribute: 'ThermalManagement' does not exist!
```

Thanks

**Comment by EthanShoeDev** [comment]
This seems to suggest that the `DellSmbios` library doesn't include these thermal settings on your laptop.

I didn't do any other testing on other machines than my own so I really can't be sure how the dell package works on your machine.

I will say, I also have the ability to change thermal settings within my UEFI bios. Maybe you could try that?

In the future I will update the app to check for this case and let the user know whether or the app will work (instead of silently erroring)

Thank you for the report!

**Comment by mll0** [comment]
Yes, I guess your'e right. And no option on the UEFI BIOS, on which I peeled my eyes.
The Dell Command Power Manager app seems to use another mysterious setting.
